web/models/respondent.ex

```diff
@@ -293,4 +293,9 @@ defmodule Ask.Respondent do
     not_ignored = fn value -> String.upcase(value) not in ignored_values end
     quiz_step["relevant"] && not_ignored.(response.value)
   end
+
+  def mobile_web_url(respondent_id) do
```

Comment:
It looks weird for me that this function is in the `Respondent` module. 
Perhaps belongs somewhere like `web/controllers/controller_helper` ?

Reply:
I moved it back to its original place. I extracted it because I planned to expose it from the API but it didn't finally happen. So there's no need to place it in the Respondent (or any other) module. Thanks!